src/software/ai/navigator/obstacle/robot_navigation_obstacle_factory.cpp

```diff
@@ -107,29 +107,24 @@ RobotNavigationObstacleFactory::createObstaclesFromMotionConstraint(
                     createFromShape(Circle(world.ball().position(), 0.5)));
             }
             break;
+
```

Comment:
Super small change but maybe remove this extra line to stay consistent with spacing throughout the rest of the file.